benchmark/misc/freelist.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var common = require('../common.js');
+var FreeList = require('freelist').FreeList;
+
+var bench = common.createBenchmark(main, {
+  n: [10000]
+});
+
+function main(conf) {
+  var n = conf.n;
+  var list = new FreeList('test', n, Object);
+  var i;
+  var used = [];
+
+
+  // First, alloc N items
+  for (i = 0; i < n; i++) {
```

Comment:
This benchmark doesn't really handle the common case (and exits way too fast). Do you think you could change it so it allocates / frees / allocates in blocks of say 500, all the way up to `n`? We try to have benchmarks take about 5-10 seconds to execute so numbers don't jump around as much.